### PR TITLE
Make sqlite instructions per yield configurable

### LINKED ISSUE #4586: Make the yielding sql instructions configurable
## Abstract

While developing 1.25.0 we saw some crashes of the backend due to sqlite yielding being used in a wrong way and yielding during a write excecution, then removing the callback and then resuming later.

This was fixed with the critical section, but there could be areas where we do not use it properly and still may lead to a crash.

Some users say they still see crashes from time to time.

To figure out what's wrong we should make debug versions of the app and send it to the users to provide us with logs.

But also as a quick/easy fix we should allow them to configure the sqlite vm callback. Setting to 0, disables it.

## Task

- Make the sqlite callback yielding configurable. https://github.com/rotki/rotki/blob/185b6d19f07a0c9f611539bc8267d468af7dd1f6/rotkehlchen/db/drivers/gevent.py#L25
- That means making it a console argument, passing it over to all places in the DB where it's used and using it.
- The console argument should be set by the frontend via the cogwheel like the other console arguments (log version, data directory etc)

### PR DESCRIPTION
**Pull request by nebolax**
Closes #4586

### CODE REVIEW

**Comment by kelsos** [comment]
@LefterisJP frontend looks good, so if you find things fine on your side you can go ahead and merge.

File: rotkehlchen/tests/conftest.py [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
+from rotkehlchen.args import DEFAULT_SQL_VM_INSTRUCTIONS_CB
 from rotkehlchen.config import default_data_directory
 from rotkehlchen.globaldb.handler import GlobalDBHandler
 from rotkehlchen.logging import TRACE, add_logging_level, configure_logging
 from rotkehlchen.tests.utils.args import default_args
 
 add_logging_level('TRACE', TRACE)
 configure_logging(default_args())
-GlobalDBHandler(default_data_directory())
+GlobalDBHandler(default_data_directory(), DEFAULT_SQL_VM_INSTRUCTIONS_CB)
```
Comment (LefterisJP) [inline rotkehlchen/tests/conftest.py:18]: Add a comment here that this is customizable for the globaldb during the tests through this fixture:

https://github.com/rotki/rotki/blob/eb5bef269207e8b84075ee36ce7c3804115ed6a0/rotkehlchen/tests/fixtures/globaldb.py#L33

File: tools/uniswapv2/detect.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def read_file_if_exists(name: str) -> Optional[Dict[str, Any]]:
         assert DB_USERNAME_VAR in os.environ, f'Missing {DB_USERNAME_VAR} in the env vars'
         assert DB_PASSWORD_VAR in os.environ, f'Missing {DB_PASSWORD_VAR} in the env vars'
         db_username, db_password = os.environ.get(DB_USERNAME_VAR), os.environ.get(DB_PASSWORD_VAR)
-        database = DBHandler(db_username, db_password, msg_aggregator, None)
+        database = DBHandler(db_username, db_password, msg_aggregator, None, 5000)
```
Comment (LefterisJP) [inline tools/uniswapv2/detect.py:207]: I know it's old file, but please add kwargs here

**Review by LefterisJP — COMMENTED** [review]
Some easy comments to handle

**Review by LefterisJP — APPROVED** [review]
LGTM since I implemented the small comments I had

**Pull request merged**